webdriver/tests/new_session/async.py

```diff
@@ -0,0 +1,25 @@
+# META: timeout=long
+
+import pytest
+
+from .conftest import product, flatten
+
+from tests.support.asserts import assert_success
+from tests.new_session.support.create import valid_data
+
+
+@pytest.mark.parametrize("key,value", flatten(product(*item) for item in valid_data))
+def test_valid(new_session, add_browser_capabilities, key, value):
+    new_async_session_response, get_session_response = new_async_session(
```

Comment:
In the spec you define that as a new command with a different end-point. As such we should move related tests to a new sub directory called `new_async_session`.